App/KeychainWrapper/KeychainWrapperDemo/KeychainWrapperDemo/ViewModel/CustomException.swift

```diff
@@ -0,0 +1,19 @@
+enum CustomException: Error {
+    case setDataToKeychain(message: String)
+    case getDataFromKeychain(message: String)
+    case cannotConvertStringToData(message: String)
+    case cannotDeleteDataFromGivenKey(message: String)
+    
+    var description: String {
```

Comment:
We can try this (swift-case-paths)[https://github.com/pointfreeco/swift-case-paths] lib for getting the message. I love this kind of helper method but it may be tedious to add them by hand :) Your call :)

Reply:
Don't know is it worth effort, since this is demo app for wrapper testing purposes.

Reply:
It's all to have a pretext to learn something new :D